src/card/card-header.jsx

```diff
@@ -109,12 +109,18 @@ const CardHeader = React.createClass({
       avatar = <Avatar src={this.props.avatar} style={styles.avatar} />;
     }
 
+    const {
+      title,
+      subtitle,
+      ...attrs,
```

Comment:
Nitpick: Can you rename this to `...other`, we general use that name for remaining spread attributes...otherwise looks good :+1: